File: src/_1ProgrammingBasicsWithJavaMay2023/_3ConditionalStatementsAdvanced/_1Lab/_11FruitShop.java
package _1ProgrammingBasicsWithJavaMay2023._3ConditionalStatementsAdvanced._1Lab;

import java.util.Scanner;

public class _11FruitShop {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String fruit = scanner.nextLine();
        String dayOfWeek = scanner.nextLine();
        double quantity = Double.parseDouble(scanner.nextLine());
        boolean isWorkingDay = dayOfWeek.equals("Monday")
                || dayOfWeek.equals("Tuesday")
                || dayOfWeek.equals("Wednesday")
                || dayOfWeek.equals("Thursday")
                || dayOfWeek.equals("Friday");
        boolean isWeekend = dayOfWeek.equals("Saturday")
                || dayOfWeek.equals("Sunday");
        boolean isValid = true;
        double price = 0;
        if (isWorkingDay) {
            if (fruit.equals("banana")) {
                price = 2.50;
            } else if (fruit.equals("apple")) {
                price = 1.20;
            } else if (fruit.equals("orange")) {
                price = 0.85;
            } else if (fruit.equals("grapefruit")) {
                price = 1.45;
            } else if (fruit.equals("kiwi")) {
                price = 2.70;
            } else if (fruit.equals("pineapple")) {
                price = 5.50;
            } else if (fruit.equals("grapes")) {
                price = 3.85;
            } else {
                isValid = false;
            }
        } else if (isWeekend) {
            if (fruit.equals("banana")) {
                price = 2.70;
            } else if (fruit.equals("apple")) {
                price = 1.25;
            } else if (fruit.equals("orange")) {
                price = 0.90;
            } else if (fruit.equals("grapefruit")) {
                price = 1.60;
            } else if (fruit.equals("kiwi")) {
                price = 3.00;
            } else if (fruit.equals("pineapple")) {
                price = 5.60;
            } else if (fruit.equals("grapes")) {
                price = 4.20;
            } else {
                isValid = false;
            }
        } else {
            isValid = false;
        }
        if (isValid) {
            double result = quantity * price;
            System.out.printf("%.2f", result);
        } else {
            System.out.println("error");
        }
    }
}

File: src/_1ProgrammingBasicsWithJavaScriptNovember2024/_3ConditionalStatementsAdvanced/_1Lab/_11FruitShop.js
function fruitShop(fruit, dayOfWeek, quantity) {
    let fruitPrice = 0;
    if (dayOfWeek === "Monday" || dayOfWeek === "Tuesday" || dayOfWeek === "Wednesday" || dayOfWeek === "Thursday" || dayOfWeek === "Friday") {
        if (fruit === "banana") {
            fruitPrice = 2.50;
        } else if (fruit === "apple") {
            fruitPrice = 1.20;
        } else if (fruit === "orange") {
            fruitPrice = 0.85;
        } else if (fruit === "grapefruit") {
            fruitPrice = 1.45;
        } else if (fruit === "kiwi") {
            fruitPrice = 2.70;
        } else if (fruit === "pineapple") {
            fruitPrice = 5.50;
        } else if (fruit === "grapes") {
            fruitPrice = 3.85;
        } else {
            console.log(`error`);
            return;
        }
    } else if (dayOfWeek === "Saturday" || dayOfWeek === "Sunday") {
        if (fruit === "banana") {
            fruitPrice = 2.70;
        } else if (fruit === "apple") {
            fruitPrice = 1.25;
        } else if (fruit === "orange") {
            fruitPrice = 0.90;
        } else if (fruit === "grapefruit") {
            fruitPrice = 1.60;
        } else if (fruit === "kiwi") {
            fruitPrice = 3.00;
        } else if (fruit === "pineapple") {
            fruitPrice = 5.60;
        } else if (fruit === "grapes") {
            fruitPrice = 4.20;
        } else {
            console.log(`error`);
            return;
        }
    } else {
        console.log(`error`);
        return;
    }

    const sum = quantity * fruitPrice;

    console.log(`${sum.toFixed(2)}`);
}